src/operator/distinct.rs

```diff
@@ -48,6 +51,15 @@ where
     /// This is equivalent to `self.integrate().distinct().differentiate()`, but
     /// is more efficient.
     pub fn distinct_incremental(&self) -> Stream<Circuit<P>, Z>
+    where
```

Comment:
should the above comment move to the _inner function?

Reply:
`_inner` is not `pub`, so it won't be visible there.